shared/common-adapters/floating-picker.js.flow

```diff
@@ -0,0 +1,21 @@
+// @flow
+import * as React from 'react'
+
+// Mobile only. Show native dropdown UI.
+
+export type PickerItem<T> = {|label: string, value: T|}
+
+export type Props<T: string | number> = {
```

Comment:
Did some work typing this; now if you pass in string `PickerItem.value`s, flow will know that `onSelect` will be called with a string (and the same for number)